src/test/java/hooks/TestHooks.java

```diff
@@ -0,0 +1,34 @@
+package hooks;
+
+import org.openqa.selenium.WebDriver;
+
+import cucumber.api.java.After;
+import cucumber.api.java.Before;
+import dataProviders.ConfigFileReader;
+import managers.WebDriverManager;
+import stepDefinition.AbstractStepDef;
+
+public class TestHooks extends AbstractStepDef {
+
+    private ConfigFileReader fileReader;
+
+    private WebDriverManager webDriverManager;
+
+    private static void setDriver(WebDriver driver) {
+        webDriver = driver;
+    }
+
+    @Before
+    public void launchChromeDriver() {
+        webDriverManager = new WebDriverManager();
+        fileReader = new ConfigFileReader();
```

Comment:
this action does not belongs to this method, you can do this when you initiate this